LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/LoRaMessage/LoRaPayloadJoinAccept.cs

```diff
@@ -195,8 +195,10 @@ public override byte[] PerformEncryption(AppKey key)
 
         public override byte[] GetByteMessage()
```

Comment:
nit: could turn this into a one-liner, skipping the list:

```suggestion
        public override byte[] GetByteMessage() => RawMessage.Prepend((byte)MHdr).ToArray();
```

Reply:
Sorry, but this should have never been a _suggested change that could be applied_ immediately. On GitHub, I couldn't somehow mark the suggestion as a replacement for the whole function, but then forgot to revert the suggestion into a regular code snippet before posting my comment. In short, [this has broken the build](https://github.com/Azure/iotedge-lorawan-starterkit/runs/4760665538?check_suite_focus=true#step:8:36).